Kajabi/sage-lib
# Form Select - added rails component and updated docs

## Description

- [x] - updated docs and props

## Test notes

Visit 

### Steps for testing
1. Visit rails form select page: http://localhost:4000/pages/element/form_select
2. Verify demo and docs
